internal/services/hybridkubernetes/registration.go

```diff
@@ -0,0 +1,29 @@
+package hybridkubernetes
+
+import "github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+
+type Registration struct{}
+
+// Name is the name of this Service
+func (r Registration) Name() string {
+	return "HybridKubernetes"
+}
+
+// WebsiteCategories returns a list of categories which can be used for the sidebar
+func (r Registration) WebsiteCategories() []string {
+	return []string{
+		"HybridKubernetes",
```

Comment:
per the product name, is this Azure Arc? https://docs.microsoft.com/en-us/azure/azure-arc/kubernetes/

```suggestion
		"Arc",
```

Reply:
Will rename it to "ArcKubernetes"